tools/README.md

```diff
@@ -1,42 +1,44 @@
 # tools
-## parser.py
-Usage:
+## mdParser.js
+### Mission
+Parses the README file to find archives and events. extract every events from those markdown files and saves them in a `all-event.json` file
+### Usage
 ```
-python3 parser.py <year> [outputFile="-"]
-# Special use case, parse README.md from root folder
-python3 parser.py README [outputFile="-"]
+node mdParser.js 
 ```
 outputFile whose value is "-" are going to stdout.
 
-#### Example Output
+### Event Output format
 ```json
 [
 	{
 		"name": "DDD Europe",
 		"date": [
-			1612371600,
-			1612458001
+			1612371600000,
+			1612458001000
 		],
 		"hyperlink": "https://dddeurope.com/2021/handson-conference/",
 		"location": "Online (les autres annees Amsterdam, Pays-bas)",
-		"misc": ""
+		"misc": "<a href=\"https://oredev.org/callforpaper\"><img alt=\"CFP Oredev malmo 2023\" src=\"https://img.shields.io/static/v1?label=CFP&message=until%2031-March-2023&color=red\"></a>",
+		"cfp":{
+			"link":"https://oredev.org/callforpaper",
+			"until":"31-March-2023",
+			"untilDate":1704409200000
+		},
+		"status":"open"
```

Comment:
Is it the status of a CFP or a conference? This status should always be equals to "open"?
Thx :)

Reply:
it's the status of the conference.
I defaulted to 'open', but it takes the value expressed before the date

eg: "* [Pixelated] 04-05: myconf ..." --> status = pixelated

BTW, it appears mostly in 2020.md file ... because of covid, I guess

Reply:
Thanks, I understand and yes we can keep it on order, for example, to add the "canceled" status possibility :)